com.warlordsoftwares.formatlab.yml

```diff
@@ -0,0 +1,169 @@
+app-id: com.warlordsoftwares.formatlab
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: formatlab
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=host
+  - --filesystem=xdg-download
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-run/gvfsd
+  - --device=dri
+  - --socket=pulseaudio
+  - --env=QT_QPA_PLATFORM=wayland
+
+tags:
+  - proprietary
+
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /bin/pyrcc5
+  - /bin/pyuic5
+
+modules:
+  - name: x264
+    config-opts:
+      - '--disable-cli'
+      - '--enable-shared'
+    sources:
+      - type: git
+        url: https://code.videolan.org/videolan/x264.git
+        commit: 5db6aa6cab1b146e07b60cc1736a01f21da01154
+
+  - name: x265
+    buildsystem: cmake-ninja
+    builddir: true
+    subdir: source
+    config-opts:
+      - '-DCMAKE_BUILD_TYPE=Release'
+      - '-DENABLE_CLI=OFF'
+    sources:
+      - type: archive
+        url: https://deb.debian.org/debian/pool/main/x/x265/x265_3.5.orig.tar.gz
+        sha256: e70a3335cacacbba0b3a20ec6fecd6783932288ebc8163ad74bcc9606477cae8
+
+  - name: libass
+    sources:
+      - type: archive
+        url: https://github.com/libass/libass/releases/download/0.16.0/libass-0.16.0.tar.xz
+        sha256: 5dbde9e22339119cf8eed59eea6c623a0746ef5a90b689e68a090109078e3c08
+      - type: script
+        commands:
+          - autoreconf -fiv
+        dest-filename: autogen.sh
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+      - /share
+      - '*.a'
+      - '*.la'
+
+  - name: luajit
+    no-autogen: true
+    make-install-args:
+      - PREFIX=${FLATPAK_DEST}
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /share/man
+    sources:
+      - type: git
+        url: https://github.com/LuaJIT/LuaJIT.git
+        #branch: v2.1
+        commit: 4c2441c16ce3c4e312aaefecc6d40c4fe21de97c
+
+  - name: mpv
+    buildsystem: simple
+    build-commands:
+      - python3 waf configure --disable-alsa --disable-build-date --disable-manpage-build
```

Comment:
Do you need mov the player or just libmpv?

Reply:
i need full mpv player support for the app